bin/brew

```diff
@@ -84,7 +84,18 @@ HOMEBREW_LIBRARY="${HOMEBREW_REPOSITORY}/Library"
 
 # Copy and export all HOMEBREW_* variables previously mentioned in
 # manpage or used elsewhere by Homebrew.
-for VAR in BAT_THEME BROWSER DISPLAY EDITOR NO_COLOR TMUX DBUS_SESSION_BUS_ADDRESS XDG_RUNTIME_DIR
+PASSTHROUGH_VARS=(
```

Comment:
```suggestion
HOMEBREW_COPY_VARS=(
```

or something because these are not passed through but `ENV_VAR_NAMES` (which could get a rename below and reformatting too) are.